circle-mlir/CMakeLists.txt

```diff
@@ -42,4 +42,6 @@ if(CIRCLE_MLIR_LOCALINST)
   message(STATUS "CIRCLE_MLIR_LOCALINST=${CIRCLE_MLIR_LOCALINST}")
 endif()
 
+set(RES_CIRCLE_SCHEMA "${CMAKE_SOURCE_DIR}/../res/CircleSchema")
```

Comment:
I have a question.
We cannot use "${NNAS_PROJECT_SOURCE_DIR}" like the compiler instead of "${CMAKE_SOURCE_DIR}/.."?

Reply:
this project has no relation with existing CMake files; not to make dependency.

Reply:
Thanks for your information.